src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/UnixExportProvider.cs

```diff
@@ -197,25 +197,26 @@ private byte[] ExportPfx(SafePasswordHandle password)
                 attrBytes[1] = sizeof(int);
                 MemoryMarshal.Write(attrBytes.AsSpan(2), ref keyIdx);
 
+                AttributeAsn attribute = new AttributeAsn
```

Comment:
Why did this need to move out?

Reply:
Ah, presumably to avoid nullable annotations in the indirect reference it used to have.